index.en.html

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
Let's move that to `en/index.html`, so we can:

1. Refer to `beta.gouv.fr/en` rather than `beta.gouv.fr/index.en.html`.
2. Slowly build up content in English in a scoped directory.